scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -365,8 +374,42 @@ def config_lvm(self):
     # Check if cvg is non empty
     if not cvg:
         raise MotrError(errno.EINVAL, "cvg is empty\n")
+    return cvg_cnt, cvg
 
+def update_bgsize(self):
     dev_count = 0
+    lvm_min_size = None
+    lvm_min_size = None
+
+    cvg_cnt, cvg = get_cvg_cnt_and_cvg(self)
+    for i in range(int(cvg_cnt)):
+        cvg_item = cvg[i]
+        try:
+            metadata_devices = cvg_item["metadata_devices"]
+        except:
+            raise MotrError(errno.EINVAL, "metadata devices not found\n")
+        check_type(metadata_devices, list, "metadata_devices")
+        sys.stdout.write(f"\nlvm metadata_devices: {metadata_devices}\n\n")
+        for device in metadata_devices:
+            cmd = f"pvs --noheadings {device}"
+            vgname = (execute_command(self, cmd)[0]).split(sep=None)[1]
+            cmd = "lvdisplay | grep \"LV Path\" | grep {} | grep -v swap".format(vgname)
+            lv_list = (execute_command(self, cmd)[0]).replace("LV Path", '').split('\n')[0:-1]
+            for i in range(len(lv_list)):
+                lv_list[i] = "".join(lv_list[i].split())
+                lv_path = lv_list[i]
+                lvm_min_size = calc_lvm_min_size(self, lv_path, lvm_min_size)
+    if lvm_min_size:
+        sys.stdout.write(f"setting MOTR_M0D_IOS_BESEG_SIZE to {lvm_min_size}\n")
+        cmd = f'sed -i "/MOTR_M0D_IOS_BESEG_SIZE/s/.*/MOTR_M0D_IOS_BESEG_SIZE={lvm_min_size}/" {MOTR_SYS_CFG}'
+        execute_command(self, cmd)
+
+def config_lvm(self):
+    dev_count = 0
+    lvm_min_size = None
+    lvm_min_size = None
```

Comment:
Duplicate lines are here.

Reply:
Done 683d315

Reply:
This duplication is still there.
Please check and remove.